### DISCUSSION TITLE
can i create google meet ?

### DISCUSSION THREAD

**Discussion by ultimate-solutions-user2** [spatie/laravel-google-calendar#157] [Q&A]

**Comment by marcorivm** [comment]
@ultimate-solutions-user2 did you get it ?

**Comment by joefreire** [comment]
@ultimate-solutions-user2 I don't know if my PR can use in this excellet project but I create a fork solution to create a conference and generate the link automatically on google meet

**Reply by jcorrego** [reply]
Anyone was able to get this working? @joefreire I added also the code to my implementation but it stills doesn't work... any advice?
